pkg/ebpftracer/c/tracee.bpf.c

```diff
@@ -1309,6 +1309,14 @@ int tracepoint__sched__sched_process_exec(struct bpf_raw_tracepoint_args *ctx)
         {
             flags |= FS_EXE_UPPER_LAYER;
         }
+
+        struct task_struct *task = (struct task_struct *) bpf_get_current_task();
+        struct file *exe = get_exe_file(task);
```

Comment:
What is the difference here from call above which also gets a file `struct file *file = get_file_ptr_from_bprm(bprm);` ?

Reply:
I thought it might make a difference, but on closer inspection, it should be the same (at least for `execve`s).